src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final int OPERATION_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+
+    private String readFromFile(String fromFileName) {
+
+        StringBuilder stringBuilder = new StringBuilder();
+        File fileToRead = new File(fromFileName);
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileToRead))) {
+            String line = bufferedReader.readLine();
+            stringBuilder.append(line);
+            while (line != null) {
+                line = bufferedReader.readLine();
+                stringBuilder.append(" ").append(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Error while reading the file");
+        }
+        String[] resultingString = stringBuilder.toString().split(" ");
+        int buySum = 0;
+        int supplySum = 0;
+        String[] stringSplit;
+        for (String line : resultingString) {
+            stringSplit = line.split(",");
```

Comment:
```suggestion
        for (String line : resultingString) {
            String[] stringSplit = line.split(DELIMITER);
```
let's move comma to a constant 

and not declare variables separately, if we can declare them when we use the variable